src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -702,6 +734,9 @@ class PolicyHandler : public PolicyHandlerInterface,
 #endif  // EXTERNAL_PROPRIETARY_MODE
   bool SaveSnapshot(const BinaryMessage& pt_string, std::string& snap_path);
 
+ private:
+  static const std::string kLibrary;
```

Comment:
@LuxoftAKutsan , policy library. Dont know why it is marked as new code, but it is already in source code in extended_policy.
`const std::string PolicyHandler::kLibrary = "libPolicy.so";`